src/core/theme/reset/index.ts

```diff
@@ -24,30 +20,22 @@ const fontBase = css`
   -webkit-font-smoothing: antialiased;
 `;
 
-export const font = (props: TokensOrThemeProps) => (
+export const font = (props: SuomifiThemeProp) => (
   typographyToken: TypographyProp,
 ) => css`
   ${fontBase}
-  ${themeOrTokens(props).typography[typographyToken]}
-`;
-
-export const internalTokenFont = (
-  typographyToken: keyof internalTypographyTokensProp,
-) => css`
-  ${fontBase}
-  ${internalTypographyTokens[typographyToken]}
+  ${props.theme.typography[typographyToken]}
 `;
 
-export const input = (props: TokensOrThemeProps) => {
-  const theme = themeOrTokens(props);
+export const input = (props: SuomifiThemeProp) => {
```

Comment:
Wouldn't changing this require refactoring also the related element and font utilities? And if this could now be applied to all related references, maybe a separate issue to handle it since this PR is already quite big?

Reply:
Added to following ticket.